internal/usecase/board.go

```diff
@@ -118,3 +134,18 @@ func (usecase *usecase) shouldShowOnUserLevel(board bbs.BoardRecord, userID stri
 func getArticleURL(boardId string, filename string) string {
 	return fmt.Sprintf("https://ptt-app-dev-codingman.pichuchen.tw/bbs/%s/%s.html", boardId, filename)
 }
+
+func searchArticles(fileHeaders []bbs.ArticleRecord, cond *ArticleSearchCond) []bbs.ArticleRecord {
+	var targetArticles []bbs.ArticleRecord
+
+	for _, f := range fileHeaders {
+		if strings.Contains(strings.ToLower(f.Title()), strings.ToLower(cond.Title)) &&
+			strings.Contains(strings.ToLower(f.Owner()), strings.ToLower(cond.Author)) &&
+			((cond.RecommendCountGe == 0) ||
+				(cond.RecommendCountGe > 0 && f.Recommend() >= cond.RecommendCountGe) ||
+				(cond.RecommendCountGe < 0 && f.Recommend() <= cond.RecommendCountGe)) {
```

Comment:
因為我們有 Le 的設定，因此應該不需要 < 0 作為 Le ，另外考慮有人可能希望知道 > -20 的文章，因此建議改為以下程式。

```suggestion
				(cond.RecommendCountLe > 0 && f.Recommend() <= cond.RecommendCountLe)) {
```

Reply:
> 因為我們有 Le 的設定，因此應該不需要 < 0 作為 Le ，另外考慮有人可能希望知道 > -20 的文章，因此建議改為以下程式。

目前 Ptt 功能上是輸入負數就找尋噓文數以上的文章，這樣有需要做到 > -20 的部份嗎？還是你的 > -20 是指噓文數 > 20？
另外要做的話，以建議的程式來看是不是代表 Le 預設就是尋找噓文數？

不過 Le > 0 && Recommend <= Le 的判斷有點怪。Le > 0 會導致結果為 推/噓文數 <= 某個正數。
似乎也無法找尋 > -20 的文章，還是我有誤會你的意思＠＠

Reply:
輸入負數找噓文數以上的文章是前端的設計，後端這邊做到和文件一致應該就行了。

應該不用另外判斷 Le > 0, 但是可能要判斷 Le != nil 或是其他方式代表進來的條件有 Le